@navikt/ds-icons/figma-api/index.js

```diff
@@ -0,0 +1,71 @@
+const api = require("./api");
+const fs = require("fs");
+const path = require("path");
+const pLimit = require("p-limit");
+const kebab = require("lodash.kebabcase");
+
+const main = async () => {
+  const iconFolder = "./svg";
+  const misses = [];
+
+  // Limiting to ~20 concurrent downloads seems to lead to fewer erros
+  const limit = pLimit(20);
+
+  const iconNodesArr = await api.getNodeChildren().catch((e) => {
+    throw e;
+  });
+
+  const iconUrls = await api.getSvgImageUrls(
+    iconNodesArr.map((node) => node.node_id).join(",")
+  );
+
+  if (!fs.existsSync(iconFolder)) {
+    fs.mkdirSync(iconFolder);
+  } else {
+    fs.rmdirSync(iconFolder);
```

Comment:
Etter `rmdir` er det vel behov for `mkdir`?

Kanskje det gir mest mening å si noe sånt som: 
```
if(exists)
    rmdir
mkdir
```

Reply:
Ser ut som ` fs.writeFileSync` lager dir hvis det ikke fungerer, men fikser det for sikkerhets skyld.